terraform/production/main.tf

```diff
@@ -36,47 +36,64 @@ provider "aws" {
 }
 
 locals {
-  prefix                    = "core-prod"
+  prefix = "core-prod"
+
+  rails_env = "production"
+
   app_host                  = "submit-social-housing-data.levellingup.gov.uk"
-  app_task_desired_count    = 2
-  application_port          = 8080
-  database_port             = 5432
   load_balancer_domain_name = "lb.submit-social-housing-data.levellingup.gov.uk"
-  provider_role_arn         = "arn:aws:iam::977287343304:role/developer"
-  redis_port                = 6379
+
+  provider_role_arn = "arn:aws:iam::977287343304:role/developer"
+
+  app_task_desired_count = 4
+
+  application_port = 8080
+  database_port    = 5432
+  redis_port       = 6379
 }
 
 module "application" {
   source = "../modules/application"
 
+  app_task_cpu    = 512
+  app_task_memory = 1024
+
+  sidekiq_task_cpu           = 1024
+  sidekiq_task_desired_count = 2

```

Comment:
This seems a bit pointless, but is what's happening on paas. I suppose it protects against a task going down at just the wrong moment. Nevertheless we could probably get rid of it, but my default was to match paas.